appveyor.yml

```diff
@@ -1,6 +1,8 @@
-image: Visual Studio 2017
+image: Visual Studio 2019
```

Comment:
I'm not 100% sure docker is working in this one: https://www.appveyor.com/docs/windows-images-software/#docker ...

Reply:
Yeah, I'm at a loss how a PR can possibly pass that test 😅
What can I do, should I revert that file?

Reply:
Yeah, try to revert that file, it should go through